### ISSUE TITLE
Newrelic Infrastructure

### ISSUE THREAD

**Issue by Evanlec** [weareinteractive/ansible-newrelic#18]
Apparently Newrelic is dropping support for the nrsysmon in favor of the new "Infrastructure" platform.

Any chance we could get a ansible role for this?

**Comment by franklinkim** [comment]
hi, haven't looked into that. Do you have a link?

**Comment by caseybecking** [comment]
https://newrelic.com/infrastructure

**Comment by franklinkim** [comment]
thx, will take a look at this

**Comment by DimmKirr** [comment]
@Evanlec Hi, any link to the source of this information? Just curious?

Thanks

**Comment by Evanlec** [comment]
https://docs.newrelic.com/docs/servers/servers-pages/servers-ui/new-relic-servers-new-relic-infrastructure

Accounts created after Nov 2016 will not have access to the "Servers" module. Only Infrastructure. Which tells me they're phasing out Servers in favor of Infra.

**Comment by DimmKirr** [comment]
Thank you.

Looks like they are killing free server monitoring completely?
Infrastructure doesn't seem to be free, or am i missing something?

On Thu, Jan 26, 2017 at 11:46 AM Evan LeCompte <notifications@github.com>
wrote:

https://docs.newrelic.com/docs/servers/servers-pages/servers-ui/new-relic-servers-new-relic-infrastructure

Accounts created after Nov 2016 will not have access to the "Servers"
module. Only Infrastructure. Which tells me they're phasing out Servers in
favor of Infra

-- 
Sincerely,
  Dmitry Kireev
  Cell: +1 (818) 850-55-77
  Skype: kireev.co

**Comment by spikerobinson** [comment]
Yeah it's discouraging. We had Servers first, tried Infrastructure, went back to Servers. Infrastructure was at best alpha software. But they are pushing it hard. Not sure what happened to the idea that server coverage was a freebie to get you to buy their APM. 

By the way you can install Infrastructure across your estate with a one line ansible ad-hoc command. :-)

**Comment by DimmKirr** [comment]
We are probably just going to use TICK stack, works well so far.

On Mon, Feb 20, 2017 at 4:00 PM, spikerobinson <notifications@github.com>
wrote:

> hard





-- 
Sincerely,
  Dmitry Kireev
  Cell: +1 (818) 850-55-77
  Skype: kireev.co
